configs/_base_/datasets/imagenet_bs64_mixer_224.py

```diff
@@ -0,0 +1,48 @@
+# dataset settings
+dataset_type = 'ImageNet'
+
+# change according to https://github.com/rwightman/pytorch-image-models/blob
+# /master/timm/models/mlp_mixer.py
+img_norm_cfg = dict(
+    mean=[127.5, 127.5, 127.5], std=[127.5, 127.5, 127.5], to_rgb=True)
+
+# training is not supported for now
+train_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(type='RandomResizedCrop', size=224, backend='cv2'),
+    dict(type='RandomFlip', flip_prob=0.5, direction='horizontal'),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='ImageToTensor', keys=['img']),
+    dict(type='ToTensor', keys=['gt_label']),
+    dict(type='Collect', keys=['img', 'gt_label'])
+]
+
+test_pipeline = [
+    dict(type='LoadImageFromFile'),
+    dict(
+        type='Resize', size=(256, -1), backend='cv2', interpolation='bicubic'),
+    dict(type='CenterCrop', crop_size=224),
+    dict(type='Normalize', **img_norm_cfg),
+    dict(type='ImageToTensor', keys=['img']),
+    dict(type='Collect', keys=['img'])
+]
+data = dict(
+    samples_per_gpu=32,
```

Comment:
Whether the `samples_per_gpu` is 32 or 64?

Reply:
It should be 64, fixed.